static/app/utils/replays/timer.tsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * EventTarget has ~97% browser support
+ */
+export class Timer extends EventTarget {
+  private _id: number | null = null;
+  private _active: boolean = false;
+  private _start: number = -1;
```

Comment:
why is start -1?

Reply:
Ah this was used to signify that it's been initialized but not started, but since there's an `_active` flag, this isn't needed and can safely be init'd to 0